recipe/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+# qtpdf
```

Comment:
included by mistake?

Reply:
No, it's from conda-forge. All these packages are required to run `qtpdf` or `webpdf` correspondingly. We do not use yum_requirements.txt, but they often happen in conda-forge recipes